package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "malaysia_vacination_metrics",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.9.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
```

Comment:
- [ ] [Optional] You don't need to put these packages in project's dependencies, they for development environment, just put them in devDependencies